core-ui/src/shared/components/PodTemplate/PodTemplate.scss

```diff
@@ -0,0 +1,22 @@
+.template-table {
+  margin-bottom: 10px;
+
+  tr {
+    text-align: left;
+    border-bottom: 2px solid var(--sapGroup_ContentBorderColor, #d9d9d9);
+  }
+
+  th {
+    background-color: var(--sapNeutralBackground);
```

Comment:
Maybe it would look better to stick to the same convention as regular tables, so to have headers background color as `var(--sapList_HeaderBackground)`